deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml

```diff
@@ -57,15 +57,15 @@ spec:
     imagePullPolicy: IfNotPresent
   - name: csi-provisioner
     repository: k8s.gcr.io/sig-storage/csi-provisioner
-    tag: "v2.1.0"
+    tag: "v2.2.0"
     imagePullPolicy: IfNotPresent
   - name: csi-attacher
     repository: k8s.gcr.io/sig-storage/csi-attacher
     tag: "v3.1.0"
     imagePullPolicy: IfNotPresent
   - name: csi-snapshotter
     repository: k8s.gcr.io/sig-storage/csi-snapshotter
-    tag: "v3.0.2"
+    tag: "v3.0.3"
```

Comment:
why not v4.0.0?

Reply:
Bumping to v4.x requires having an external snapshotter. We can't force it.

Reply:
we have a ticket for it (CSI-2873) when the time comes